### DISCUSSION TITLE
Key bindings with different layout

### DISCUSSION THREAD

**Discussion by b1ru** [stumpwm/stumpwm#1116] [Q&A]
Is it possible to use the default key bindings, with a different layout   - in my case, Greek - ? 
Something like the reverse-im-mode in emacs.

**Comment by dmb2** [comment]
I'm not sure... It might be possible to hack together a translation map between the keysyms stumpwm sees and a normal qwerty layout.  An immediate problem is that stumpwm doesn't know about international input and it is non-trivial to add, requiring upstream modifications of clx to support xkeyboard input.  More detail here:

http://dmb2.github.io/posts/clx_extensions.html
